src/main/kotlin/com/mineinabyss/looty/ResourcepackGeneration.kt

```diff
@@ -0,0 +1,182 @@
+package com.mineinabyss.looty
```

Comment:
because of 2d items you dont always wanna specify a model. So leaving it without a default is confusing as config will cry when it isnt defined. Essentially u can use model to use a 3d model u made or just the texture and itll generate the json file to tie item to texture